integration/kafka/src/main/java/alluxio/kafka/connect/AlluxioSinkTaskConstants.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.kafka.connect;
+
+/**
+ * Constants used in alluxio sink task.
+ */
+public class AlluxioSinkTaskConstants {
+
+  public static final int OFFSET_LENGTH = 10;
```

Comment:
I feel these two public constants can be defined inside `AlluxioSinkTask`.
this is not critical, up to you